### PR TITLE
Fixes and Rev4 updates for ssh controls

### PR DESCRIPTION
**Pull request by samcornwell**
These are all the changes we have made in our repo at https://github.com/djhaynes/inspec-profile-disa_stig-el7 relating to ssh controls. This includes updates of these controls to RHEL7 STIG Rev4, as well as various fixes.

### CODE REVIEW

File: controls/V-71939.rb [hunk 1]
```diff
@@ -1,62 +1,45 @@
...
+  tag "fix_id": "F-78291r2_fix"
+  # TODO: We should not allow a nil or unset value - as someday the defualt may change.
+  # TODO: Require that user be perscriptive about the state they want - yes or no.
 
   describe.one do
-    # case where value no line is returned
+    # case where value no line is returned ( i.e. unset or commented out )
     describe sshd_config do
-      its('PermitEmptyPasswords') { should cmp nil}
+      its('PermitEmptyPasswords') { should cmp nil }
```
Comment (trevor-vaughan) [inline controls/V-71939.rb]: This should explicitly be `no`

File: controls/V-71957.rb [hunk 2]
```diff
@@ -54,8 +34,9 @@
 PermitUserEnvironment no
 
 The SSH service must be restarted for changes to take effect."
-
+  tag "fix_id": "F-78309r2_fix"
+  # the `i` will ignore case
   describe sshd_config do
-    its('PermitUserEnvironment') { should eq 'no' }
+    its('PermitUserEnvironment') { should match %r{no}i }
```
Comment (trevor-vaughan) [inline controls/V-71957.rb]: Please revert

File: controls/V-71959.rb [hunk 3]
```diff
@@ -1,61 +1,41 @@
...
 \"HostbasedAuthentication\" keyword and set the value to \"no\":
 
 HostbasedAuthentication no
 
 The SSH service must be restarted for changes to take effect."
-
+  tag "fix_id": "F-78311r3_fix"
   describe sshd_config do
-    its('HostbasedAuthentication') { should eq 'no' }
+    its('HostbasedAuthentication') { should match %r{no}i }
```
Comment (trevor-vaughan) [inline controls/V-71959.rb]: Please revert

File: controls/V-72237.rb [hunk 4]
```diff
@@ -1,75 +1,54 @@
...
-uri: http://iase.disa.mil
------------------
-=end
-
-client_alive_interval = attribute(
-  'client_alive_interval',
-  default: '600',
-  description: "Value expected for ClientAliveInterval in sshd_config"
-)
+CLIENT_ALIVE_INTERVAL = attribute('client_alive_interval', default: '600',
```
Comment (trevor-vaughan) [inline controls/V-72237.rb]: No globals

**Comment by samcornwell** [comment]
@trevor-vaughan Ok I addressed your requests.

**Pull request merged**